emails/beneficiary_booking_confirmation.py

```diff
@@ -18,29 +19,31 @@ def retrieve_data_for_beneficiary_booking_confirmation_email(booking: Booking) -
     is_physical_offer = ProductType.is_thing(name=offer.type) and not is_digital_offer
     is_event = ProductType.is_event(name=offer.type)
 
-    booking_date_in_tz = utc_datetime_to_dept_timezone(booking.dateCreated, venue.departementCode)
+    department_code = venue.departementCode if not is_digital_offer else beneficiary.departementCode
+    booking_date_in_tz = utc_datetime_to_dept_timezone(booking.dateCreated, department_code)
 
     beneficiary_email = beneficiary.email if feature_send_mail_to_users_enabled() else DEV_EMAIL_ADDRESS
     beneficiary_first_name = beneficiary.firstName
     formatted_booking_date = get_date_formatted_for_email(booking_date_in_tz)
     formatted_booking_time = get_time_formatted_for_email(booking_date_in_tz)
+
     offer_name = offer.name
     offerer_name = venue.managingOfferer.name
     formatted_event_beginning_time = ''
     formatted_event_beginning_date = ''
-    stock_price = stock.price
+    stock_price = str(stock.price) if stock.price > 0 else 'Gratuit'
```

Comment:
Je pense qu'il faut gérer l'offre duo
```suggestion
    stock_price = str(stock.price * booking.quantity) if stock.price > 0 else 'Gratuit'
```